firebase-appdistribution-api/src/main/java/com/google/firebase/appdistribution/FirebaseAppDistribution.java

```diff
@@ -179,6 +182,76 @@ public interface FirebaseAppDistribution {
    */
   void startFeedback(@NonNull CharSequence infoText, @Nullable Uri screenshot);
 
+  /**
+   * Displays a notification that, when tapped, will take a screenshot of the current activity, then
+   * start a new activity to collect and submit feedback from the tester along with the screenshot.
+   *
+   * <p>On Android 13 and above, this method requires the <a
+   * href="https://developer.android.com/develop/ui/views/notifications/notification-permission">runtime
+   * permission for sending notifications</a>: {@code POST_NOTIFICATIONS}. If your app targets
+   * Android 13 (API level 33) or above, you should <a
+   * href="https://developer.android.com/training/permissions/requesting">request the
+   * permission</a>.
+   *
+   * <p>When the notification is tapped:
+   *
+   * <ol>
+   *   <li>If the app is open, take a screenshot of the current activity
+   *   <li>If tester is not signed in, presents the tester with a Google Sign-in UI
+   *   <li>Starts a full screen activity for the tester to compose and submit the feedback
+   * </ol>
+   *
+   * <p>On Android 8 and above, the notification will be created in its own notification channel.
+   *
+   * @param infoTextResourceId string resource ID of text to display to the tester before collecting
+   *     feedback data (e.g. Terms and Conditions)
+   * @param importance the level of interruption for the feedback notification channel. Once the
+   *     channel's importance is set it cannot be changed except by the user. See {@link
+   *     NotificationChannel#setImportance}. On platforms below Android 8, the importance will be
+   *     translated into a comparable notification priority (see {@link
+   *     NotificationCompat.Builder#setPriority}).
+   */
+  void showFeedbackNotification(int infoTextResourceId, int importance);
```

Comment:
For my understanding, what will happen if called from background? do Android background restrictions apply to it?

Reply:
Good question - I just tested it and it works, even after navigating away from the app. This makes sense - notifications are actually the recommended way for apps to present UI to the user when in the background: https://developer.android.com/guide/components/activities/background-starts#display-notification